controllers/rcj_soccer_referee_supervisor/referee/referee.py

```diff
@@ -25,13 +26,27 @@ def check_progress(self):
             self.progress_chck[robot].track(pos)
 
             if not self.progress_chck[robot].is_progress(robot):
-                print(f'Robot {robot}: Lack of progress')
+                self.log.event(
+                    supervisor=self,
+                    type=GameEvents.LACK_OF_PROGRESS,
```

Comment:
Hmmm, since `type` is typed as `str`, we should probably call `GameEvents.LACK_OF_PROGRESS.value`

```python
>>> class GameEvents(Enum):
...     A = "AAA"
... 
>>> def foo(x):
...     print(x)
... 
>>> foo(GameEvents.A)
GameEvents.A
>>> foo(GameEvents.A.value)
AAA
```

Reply:
Hmm, that's true -- thanks @Adman!